README.md

```diff
@@ -218,7 +218,16 @@ go-licenses save <package> [package...] --save_path=<save_path>
 Checking for forbidden and unknown licenses usage:
 
 ```shell
-go-licenses check <package> [package...] 
+go-licenses check <package> [package...]
+```
+
+**Tip**: Usually you'll want to add
+
+* `...` to check all the packages in your repository
+* `--include_tests` to also check packages only imported by testing code (e.g., testing libraries/frameworks)
+
+```shell
+go-licenses check --include_tests github.com/your-org/your-repo/...
```

Comment:
instead of your org and your repo, just use go licenses repo as an example, that makes the example command runnable.

Also can you verify whether the command work? I am not 100% sure whether it works for absolute package paths. I know ... works for relative paths.

Reply:
> instead of your org and your repo, just use go licenses repo as an example, that makes the example command runnable.

Fixed in [a1b641b](https://github.com/google/go-licenses/pull/162/commits/a1b641bdbd3f7727d26ba621025f6cb2b9008ae3).

> Also can you verify whether the command work? I am not 100% sure whether it works for absolute package paths. I know ... works for relative paths.

Yes, from https://pkg.go.dev/cmd/go#hdr-Package_lists_and_patterns:

> net/... matches both net and packages in its subdirectories, like net/http.